### PR TITLE
Add matterverse provider

### PR DESCRIPTION
**Pull request by JiQi535**
The base URL is not yet activated. 

It will be linked to: https://optimade-matterverse-ai-43429a548b75.herokuapp.com/.

### CODE REVIEW

File: src/links/v1/providers.json
Comment (rartino) [inline src/links/v1/providers.json:127]: @JiQi535 Thanks for updating the entry! The policy for ` src/links/v1/providers.json ` is that it should link to an "index meta-database" so there is always one level of indirection to real data (which helps when providers later realize they want to expand into providing more than a single database). Hence, either just keep `"https://providers.optimade.org/index-metadbs/matterverse` in `providers.json` (which thus points to that index metadb, which then points at your URL), *or* also host an index meta-database somewhere under matterverse, e.g., `https://optimade-index.matterverse.ai`.

(Pinging @ml-evs to correct me if I'm saying anything wrong)

```suggestion
        "base_url": "https://providers.optimade.org/index-metadbs/matterverse",
```

Comment (JiQi535) [reply src/links/v1/providers.json]: Thanks for the commit suggestion and nice explanation. Now I understand the purpose of this base_url in providers.json.

**Comment by JiQi535** [comment]
A quick update: I have linked the base url to the desired domain, and it is working. https://optimade.matterverse.ai/

I believe the only step left is to rename/add some fields in our mongodb collection of matterverse.ai. Then this PR can be merged. @ml-evs Let me know if my understanding is correct. Thanks!

**Comment by ml-evs** [comment]
> A quick update: I have linked the base url to the desired domain, and it is working. https://optimade.matterverse.ai/
> 
> I believe the only step left is to rename/add some fields in our mongodb collection of matterverse.ai. Then this PR can be merged. @ml-evs Let me know if my understanding is correct. Thanks!

Hi @JiQi535, sorry for the delay here, I tried the link a while ago and saw that it was not working, and that still seems to be the case. Is the link in the PR definitely the right one?

**Comment by JiQi535** [comment]
> > A quick update: I have linked the base url to the desired domain, and it is working. https://optimade.matterverse.ai/
> > I believe the only step left is to rename/add some fields in our mongodb collection of matterverse.ai. Then this PR can be merged. @ml-evs Let me know if my understanding is correct. Thanks!
> 
> Hi @JiQi535, sorry for the delay here, I tried the link a while ago and saw that it was not working, and that still seems to be the case. Is the link in the PR definitely the right one?

Hi @ml-evs, it should be working now!

**Review by ml-evs — DISMISSED** [review]
Thanks for this @JiQi535, I think this is now ready to merge! You will then be able to check the provider dashboard for automated validator runs which might help iron out any remaining bugs.

**Comment by ml-evs** [comment]
For some reason the netlify preview isn't working here, but the content looks fine to me

**Comment by ml-evs** [comment]
We've realised that there is an SSL issue on our end which we will try to resolve before merging this PR.

**Comment by ml-evs** [comment]
IMO this is ready now that we have resolved CI issues (in #103).

**Review by ml-evs — APPROVED** [review]

**Comment by ml-evs** [comment]
Seems like there were no objections to this being merged, and it does not seem fair to hold up this PR based on the lack of consensus in https://github.com/Materials-Consortia/OPTIMADE/pull/455, so I will merge this at the end of the week unless there are any final comments.

**Pull request merged**
